person.rb

```diff
@@ -1,15 +1,16 @@
 # frozen_string_literal: true
 
-require 'securerandom'
 require_relative 'nameable'
 require_relative 'rental'
 
 class Person < Nameable
+  @@id_counter = 0
```

Comment:
- [ ] [OPTIONAL]
It will be a good thing if the ID of any given _person_ consists of positive integers.

**ID should be above zero:**
![image](https://user-images.githubusercontent.com/12745474/225799921-83b7c586-98f4-4d31-8fc8-9cf0804c954d.png)